### DISCUSSION TITLE
Foundryup Setup

### DISCUSSION THREAD

**Discussion by 0xhung295** [Cyfrin/foundry-full-course-cu#2813] [General]
Setup environment

**Comment by 0xhung295** [comment]
I'm getting a problem when trying to run this command 
''' curl -L https://foundry.paradigm.xyz | bash
'''
This is the error:
![image](https://github.com/user-attachments/assets/a1b9d914-bd7f-42ec-9da7-643530774025)

Anyone had the same problem? help^^Thanks

**Comment by 5hr1ganesh** [comment]
PowerShell uses Invoke-WebRequest instead of curl by default, and it doesn't have a -L parameter.

- First, download the script using Invoke-WebRequest
```
Invoke-WebRequest -Uri https://foundry.paradigm.xyz -OutFile foundry_installer.sh
```

- Then, execute the downloaded script using Bash
```
bash foundry_installer.sh
```

**Reply by EngrPips** [reply]
Awesome, Thanks for this heads-up.

**Reply by 0xhung295** [reply]
@5hr1ganesh
This is error when I run the command
![image](https://github.com/user-attachments/assets/dd3f2ccf-6fac-4827-b149-8376418419da)

**Reply by 0xhung295** [reply]
Maybe I need to restart one time to the it work or not

**Reply by EngrPips** [reply]
Yeah restart the system as suggested by the program.

**Reply by 5hr1ganesh** [reply]
@0xhung295 did it work?
